src/components/ManageAccountsDrawer/components/ImportAccounts.tsx

```diff
@@ -46,61 +46,90 @@ export type ImportAccountsProps = {
 }
 
 type TableRowProps = {
-  accountId: AccountId
+  accountIds: AccountId[]
   accountNumber: number
   asset: Asset
-  onAccountIdActiveChange: (accountId: AccountId, isActive: boolean) => void
+  onAccountIdsActiveChange: (accountIds: AccountId[], isActive: boolean) => void
+}
+
+type TableRowAccountProps = {
+  accountId: AccountId
+  asset: Asset
 }
 
 const disabledProps = { opacity: 0.5, cursor: 'not-allowed', userSelect: 'none' }
 
+const TableRowAccount = forwardRef<TableRowAccountProps, 'div'>(({ asset, accountId }, ref) => {
+  const translate = useTranslate()
+  const accountLabel = useMemo(() => accountIdToLabel(accountId), [accountId])
+  const pubkey = useMemo(() => fromAccountId(accountId).account, [accountId])
+  const isUtxoAccount = useMemo(() => isUtxoAccountId(accountId), [accountId])
+
+  const { data: account, isLoading } = useQuery(accountManagement.getAccount(accountId))
+
+  const assetBalancePrecision = useMemo(() => {
+    if (!account) return '0'
+    return fromBaseUnit(account.balance, asset.precision)
+  }, [account, asset.precision])
+
+  return (
+    <>
+      <Td fontWeight='bold'>
+        <Tooltip label={pubkey} isDisabled={isUtxoAccount}>
+          <div ref={ref}>
+            {isUtxoAccount ? (
+              <RawText>{`${accountLabel} ${translate('common.account')}`}</RawText>
+            ) : (
```

Comment:
Can this ever actually render if the parent `Tooltip` is disabled if `isUtxoAccount`?

Reply:
big 🧠